doc/nrf/libraries/caf/images/caf_buttons_states.svg

```diff
@@ -2,173 +2,289 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <!-- Generated by Microsoft Visio, SVG Export caf_buttons_states.svg Page-1 -->
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ev="http://www.w3.org/2001/xml-events"
-		width="7.34187in" height="5.61208in" viewBox="0 0 528.615 404.069" xml:space="preserve" color-interpolation-filters="sRGB"
```

Comment:
On diagram the arrow on button press between state resuming and state idle is inversed. Then we go from idle to resuming state.

Reply:
@divipillai we need to fix that, i didnt spot it by myself. Please fetch before you add changes as i fixed other stuff that Marek spotted.